README.md

```diff
@@ -103,6 +119,12 @@ We currently bundle the following projects in this image:
 * [amazon-cloudwatch-logs-for-fluent-bit](https://github.com/aws/amazon-cloudwatch-logs-for-fluent-bit)
 * [amazon-kinesis-streams-for-fluent-bit](https://github.com/aws/amazon-kinesis-streams-for-fluent-bit)
 
+### Using the AWS Plugins outside of a container
+
+You can use the AWS Fluent Bit plugins with [td-agent-bit](https://docs.fluentbit.io/manual/v/1.3/installation/td-agent-bit).
```

Comment:
You may have selected a version-specific link by design, but just in case: I believe you could choose to link to “latest” via https://docs.fluentbit.io/manual/installation/td-agent-bit

Reply:
Thanks! I've actually been wondering for a while how to do that... whenever I google Fluent Bit docs it sends me to specific version. And so do the links on their site (at least the ones that always catch my eye in the UI).